Orso.Arpa.Application/ProjectApplication/ProjectCreateDto.cs

```diff
@@ -1,15 +1,24 @@
+using AutoMapper;
 using FluentValidation;
+using static Orso.Arpa.Domain.Logic.Projects.Create;
 
 namespace Orso.Arpa.Application.ProjectApplication
 {
     public class ProjectCreateDto
     {
-        // ToDo: Add properties
```

Comment:
Das Todo ist noch offen, oder?

Reply:
Für den ersten Durchstich ok, danach (würde ich in einem separaten Pull Request abhandeln,) kommen dann die weiteren Properties.
Hinweis eingefügt:
// TODO: define further properties according to https://orso.atlassian.net/browse/ARPA-207

Reply:
Für den ersten Durchstich ok, danach (würde ich in einem separaten Pull Request abhandeln,) kommen dann die weiteren Properties.
Hinweis eingefügt:
// TODO: define further properties according to https://orso.atlassian.net/browse/ARPA-207